pepper-apis/src/main/resources/changelog-master.xml

```diff
@@ -242,6 +242,8 @@
     <include file="db-changes/schema/DDP-5310-create-statistics-types.xml" relativeToChangelogFile="true"/>
     <include file="db-changes/seed/add-languages-de-hi-it-ja-pl-tr.xml" relativeToChangelogFile="true"/>
     <include file="db-changes/schema/DDP-5277-revision-activity-naming-details.xml" relativeToChangelogFile="true"/>
+    <include file="db-changes/schema/DDP-5442-auth0-log-event.xml" relativeToChangelogFile="true"/>
```

Comment:
We generally append new liquibase files to the very bottom of the file so that merge conflicts are a little easier to resolve when multiple people are making liquibase changes in different tickets.  It's fine to leave this as-is...just mentioning it for future work.

Reply:
I think my line occured in the middle after updating (merging) the FB from develop.

Reply:
Fixed